paasta_tools/mesos/master.py

```diff
@@ -64,23 +65,44 @@ def key(self):
 
     @util.CachedProperty(ttl=5)
     def host(self):
-        return "{}://{}".format(self.config["scheme"], self.resolve(self.config["master"]))
+        return "{}://{}".format(
+            self.config["scheme"],
+            self.resolve(self.config["master"]),
+        )
+
+    @util.CachedProperty(ttl=5)
+    def cache_host(self):
+        host_url = urlparse(self.host)
+        host_url._replace(netloc=host_url.hostname + ':5055')
```

Comment:
I'm happy to let this go as a first pass, but I really don't want this here long term. We need something more general - could just be a 'caching_mesos_master' config value in the mesos-cli.json, or maybe we could make the caching layer noop for everything but /state.json and /frameworks?

Reply:
yeah, i agree, this is not ideal.